spec/factories/project.rb

```diff
@@ -8,8 +8,8 @@
       data_user_read_only { [] }
       data_user_read_write { [] }
       title { FFaker::Movie.title }
-      created_on { DateTime.now }
-      updated_on { DateTime.now }
+      created_on { Time.current.in_time_zone("America/New_York").iso8601 }
+      updated_on { Time.current.in_time_zone("America/New_York").iso8601 }
```

Comment:
we could try creating a project in rails, then manually creating it in medialfux with `ProjectMediaflux.create!` to see if the new date fields persist. Again, not an issue now — but if the plan is to be able to write to mediaflux in the future — then it may be later on.

Reply:
Thank you, @JaymeeH we are in the process of fixing this.